bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -202,12 +205,8 @@ public void transformInpatientClaim() throws FHIRException {
 		ItemsComponent eobSoleItem = eob.getItem().get(0);
 		Assert.assertEquals(Date.valueOf(sourceClaim.getDateAdmission()), eobSoleItem.getServicedPeriod().getStart());
 		Assert.assertEquals(Date.valueOf(sourceClaim.getDateDischarge()), eobSoleItem.getServicedPeriod().getEnd());
-		Assert.assertEquals(
-				eob.getProvider()
-						.getReference(),
-				bundle.getFhirResources().stream()
-						.filter(r -> r instanceof Practitioner).map(
-								r -> (Practitioner) r)
+		Assert.assertEquals(eob.getProvider().getReference(),
+				bundle.getFhirResources().stream().filter(r -> r instanceof Practitioner).map(r -> (Practitioner) r)
 						.filter(p -> p.getIdentifier().stream()
```

Comment:
Does the eclipse-formatter.xml profile set this preference? I think we are both running under the same profile, so if we need to change preferences further, we will probably want to document that as part of the Eclipse setup.